### DISCUSSION TITLE
Cannot dump db

### DISCUSSION THREAD

**Discussion by HCoban** [planetscale/discussion#26] [Bug Reports] [answered]
I am trying to dump my database but receiving an unknown database error.

I have a db called demo
```
$ pscale database list
  NAME   CREATED AT     UPDATED AT     NOTES
 ------ -------------- -------------- -------
  demo   22 hours ago   22 hours ago
```

I am trying to dump the db
```
$ pscale database dump demo main
Starting to dump all tables from database demo to folder /Users/halilcoban/.go/src/github.com/hcoban/staging/pscale_dump_demo_main
Error: failed to dump database: Unknown database 'demo' in vschema (errno 1105) (sqlstate HY000)
```

It created a `metadata` file under `pscale_dump_demo_main` directory but the file is empty.

**Comment by Anders-PlanetScale** [comment]
Halil, When you built your DB did  you check the include demo dataset checkbox?
I am following up with our engineering team but I think the error you are seeing here is related to how the onboarding-demo DB overwrites the default DB name, which is normally the name of the psdb database.

**Reply by HCoban** [reply]
Hello,

Thanks for your reply. Unfortunately, I do not remember if I checked the include demo dataset checkbox. But looking into DB, I do not see any data other than what I created via pscale cli.

I might have included the demo dataset in a branch other than main, but since I already deleted that branch I cannot query that branch's db.

**Reply by HCoban** [reply]
This following might be useful info:

When I list databases with mysql query `show databases`,  I see the name of the database is `onboarding-demo`. However, the name of the database when I check with `pscale database list` is `demo`. 

If I remember correctly, I set up the name `demo` when initializing the DB in the app (https://app.planetscale.com/)

**Reply by Anders-PlanetScale** [reply]
The presence of the onboarding-demo DB indicates you used the demo data-set.
I think this issue and your other [issue](https://github.com/planetscale/beta/discussions/25) are both related to using that demo-dataset. If you build a new DB without it you should not see either issue. I am opening a bug report with our engineering team to investigate further.

**Reply by HCoban** [reply]
Yes, I confirm that building a new DB without using onboarding-demo fixed both issues I was having. Thank you for looking into this.

**Comment by hallaroo** [comment, marked as answer]
Just for a status update: This issue with demo DB's should be fixed in the next release. We'll confirm once that's out. Thank you again for reporting!

**Reply by hallaroo** [reply]
And a final update here: The [version](https://github.com/planetscale/cli/releases/tag/v0.38.0) with this fix is now available. Thanks!
